AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3558,6 +3558,7 @@ bmwfaq.org#?#.Notices[-ext-has='a[href="//www.bmwfaq.org/help/cookies"]']
 !
 ! JS rules
 !
+cnet.com#%#AG_onLoad(function() { setTimeout(function() { var el = document.getElementById("onetrust-accept-btn-handler"); if(el) { el.click(); } }, 2500); });
```

Comment:
There is already similar rule:
https://github.com/AdguardTeam/AdguardFilters/blob/1e57cc7c97f989d625832917399851e2fc0b648a/AnnoyancesFilter/sections/cookies_specific.txt#L3605
so please remove this old rule or change a time after which this function is executed, if it's necessary.

Reply:
Or we can exchange this rule to something like this:
```
cnet.com#%#AG_onLoad(function(){if(-1==document.cookie.indexOf("OptanonConsent")){var a=new MutationObserver(function(){var b=document.querySelector("button#onetrust-accept-btn-handler");b&&(b.click(),setTimeout(function(){a.disconnect()},1E3))});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)}});
```

Reply:
In my opinion this rule is ok, we should exchange it to yours.

Reply:
> Or we can exchange this rule to something like this:
> 
> ```
> cnet.com#%#AG_onLoad(function(){if(-1==document.cookie.indexOf("OptanonConsent")){var a=new MutationObserver(function(){var b=document.querySelector("button#onetrust-accept-btn-handler");b&&(b.click(),setTimeout(function(){a.disconnect()},1E3))});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)}});
> ```

Perfect rule. I'll add it, thanks.